core/wiki/macros/toc-item-class.tid

```diff
@@ -0,0 +1,4 @@
+tags: $:/tags/Macro
+title: $:/core/macros/toc/item-class
+
+\define getItemClass() [all[current]] -[<tv-history-list>get[current-tiddler]]
```

Comment:
Is the purpose here to avoid repeating this filter? It's not great that it's a global macro because it isn't really useful in its own right, and the name doesn't suggest what it does. I think it would be better to just transclude the filter from the text field of a tiddler called something like `$:/core/macros/toc/item-class`?

Reply:
I did separate the filter, so other plugins can easily replace it. .. I'll create a transclusion as you suggest.